hwdb.d/60-keyboard.hwdb

```diff
@@ -75,7 +75,13 @@
 #   XKB_FIXED_VARIANT=""
 # Examples of such devices: the Yubikey or other key-code generating
 # devices.
-#
+
+# A device where the scan code to key code mapping is insufficient and
+# requires a special key code to symbol configuration may specify that with:
+#   XKB_FIXED_MODEL="xkbmodel"
+# Examples of such devices: Chromebooks where to top row is used for both
```

Comment:
argh, sorry, another typo: "the top row"

Reply:
I should be sorry for making so many typos in so little text ;)